src/Authentication/Authentication/Cmdlets/SetMgGraphOption.cs

```diff
@@ -0,0 +1,48 @@
+// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+using System.IO;
+using System.Management.Automation;
+
+namespace Microsoft.Graph.PowerShell.Authentication.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Set, "MgGraphOption", HelpUri = "")]
+    public class SetMgGraphOption : PSCmdlet
+    {
+        [Parameter]
+        public bool EnableLoginByWAM { get; set; }
+
+        protected override void BeginProcessing()
+        {
+            base.BeginProcessing();
+        }
+
+        protected override void ProcessRecord()
+        {
+            base.ProcessRecord();
+            if (EnableLoginByWAM)
+            {
+                GraphSession.Instance.GraphOption.EnableWAMForMSGraph = EnableLoginByWAM;
+                WriteObject("[Preview] Signin by Web Account Manager (WAM) is enabled.");
+            }
+            else
+            {
+                GraphSession.Instance.GraphOption.EnableWAMForMSGraph = false;
+                WriteObject("[Preview] Signin by Web Account Manager (WAM) is disabled.");
+            }
```

Comment:
Check to see if the parameter is bound on the cmdlet (customer explicitly sets -EnableLoginByWam) before modifying `GraphOption.EnableWAMForMSGraph` to avoid wrongfully setting the property to false when not specified.
```suggestion
            if (this.IsParameterBound(nameof(EnableLoginByWAM)))
            {
                GraphSession.Instance.GraphOption.EnableWAMForMSGraph = EnableLoginByWAM;
                var message = $"Signin by Web Account Manager (WAM) is {(EnableLoginByWAM ? "enabled" : "disabled")}.";
                WriteObject(message);
            }
```